### DISCUSSION TITLE
Possibility of adding two point turns.

### DISCUSSION THREAD

**Discussion by xVirusXx** [Mogli12/KeyboardSteer#826] [Ideas]
Is it possible to use guided turns to make a two point turn if checkbox is checked? I know skip lane is available but would be a nice to have. Could be based on the distance from center line to next lane center line. If > X then do 2 pt turn.

**Comment by Mogli12** [comment]
I do not fully understand what a two point turn is. Can you elaborate on it?

**Comment by xVirusXx** [comment]
Partial turn, reverse and resume. A U-turn two step U-turn essentially for
narrow headland and/or poor turn radius.

**Comment by Mogli12** [comment]
You can go reverse with lane guidance turned on. Did you try it?
